charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -511,3 +511,157 @@ instrumentation:
   #   limits:
   #     memory: "128Mi"
   #     cpu: "500m"
+
+# Bridge cluster role configuration
+bridgeClusterRole:
```

Comment:
Should it be the same as `opAMPBridge.enabled`?

Reply:
ah like rename this to be opAMPBridgeClusterRole? Probably... good call!